api/src/main/java/org/ccci/idm/user/Group.java

```diff
@@ -1,56 +1,17 @@
 package org.ccci.idm.user;
 
-import com.google.common.base.Function;
-import com.google.common.base.Predicates;
-import com.google.common.collect.FluentIterable;
-
 import javax.annotation.Nonnull;
-import javax.annotation.Nullable;
 import javax.annotation.concurrent.Immutable;
-import java.util.Arrays;
 import java.util.List;
 
 @Immutable
 public final class Group extends Dn {
     private static final long serialVersionUID = 8588784014544957895L;
 
-    private static final String LEGACY_PATH_TYPE = "ou";
-    private static final String LEGACY_NAME_TYPE = "cn";
-    private static final Function<String, Component> LEGACY_PATH = new Function<String, Component>() {
-        @Nullable
-        @Override
-        public Component apply(@Nullable final String value) {
-            return value != null ? new Component(LEGACY_PATH_TYPE, value) : null;
-        }
-    };
-
-    /**
-     * @deprecated Since 0.3.0, use {@link Group#Group(List)} instead.
-     */
-    @Deprecated
-    public Group(@Nonnull final String[] path, @Nonnull final String name) {
-        super(FluentIterable.of(path).transform(LEGACY_PATH).filter(Predicates.notNull())
-                .append(new Component(LEGACY_NAME_TYPE, name)).toList());
-    }
-
-    /**
-     * @deprecated Since 0.3.0, use {@link Group#Group(List)} instead.
-     */
-    @Deprecated
-    public Group(@Nonnull final String... path) {
-        this(Arrays.copyOf(path, path.length - 1), path[path.length - 1]);
-    }
-
```

Comment:
yeah, but I want to keep it distinct from `Dn`, `Dn` is a more abstract concept than a `Group`